src/ripple/test/jtx/impl/Env_test.cpp

```diff
@@ -561,6 +561,21 @@ class Env_test : public beast::unit_test::suite
                 );
     }
 
+    void testSig()
+    {
+        using namespace jtx;
+        Env env(*this);
+
+        env.fund(XRP(10000), "alice");
+        auto const baseFee = env.app().config().FEE_DEFAULT;
```

Comment:
The expected / canonical way to get fees out of an `Env` is `auto const baseFee = env.open()->fees().base;`.